conda-store-server/conda_store_server/worker/tasks.py

```diff
@@ -231,7 +232,7 @@ def delete_build_artifact(db: Session, conda_store, build_artifact):
         # ignore key
         conda_prefix = build_artifact.build.build_path(conda_store)
         # be REALLY sure this is a directory within store directory
-        if conda_prefix.startswith(conda_store.store_directory) and os.path.isdir(
+        if str(conda_prefix).startswith(conda_store.store_directory) and os.path.isdir(
```

Comment:
Bug fix: `PosixPath` object (`conda_prefix`) has no attribute `startswith`, so need to convert to `str` first